### PR TITLE
feat:Setup and controllers

### PR DESCRIPTION
**Pull request by taher-software**
In this branch, I've created for each required url : :100: 
- A route. :100: 
- An action in the correct controller. :100: 
- A view file. :100:

### CODE REVIEW

**Comment by oloomoses** [comment]
@Taher-web-dev, The project does not have a descriptive readme and therefore I cannot continue with the review. Kindly take a few minutes to write a descriptive readme

**Comment by taher-software** [comment]
> @Taher-web-dev, The project does not have a descriptive readme and therefore I cannot continue with the review. Kindly take a few minutes to write a descriptive readme
I apologize sir for this issues.
I've forgotten to add readme file, I usually do it in the beginning . I added it now. So please check my code and give me your valuable feedback.
Thank you !

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def show; end
+end
```
Comment (oloomoses) [inline app/controllers/users_controller.rb:3]: - [ ] Kindly add the `index` action as you have done in the `posts controller`
- [ ] Remember to add a corresponding view for the index action.

File: config/routes.rb [hunk 2]
```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root "hello#index"
+  resources :users, only: [:show] do
```
Comment (oloomoses) [inline config/routes.rb:7]: - [ ] Kindly add a route for the users `index` action. It should read `:users, only: [:show, :index]`

File: README.md [hunk 3]
```diff
@@ -1,24 +1,55 @@
-# README
+# BLOG-App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> The Blog app is classic example of a blog website. You will create a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
```
Comment (oloomoses) [inline README.md:3]: - [ ] The project description should be about what the project is about and not the requirements. Kindly update it.

**Review by oloomoses — COMMENTED** [review]
Hi @Taher-web-dev, 

**Highlights**
- Linter and rubocop setup :heavy_check_mark: 
- Descriptive PR :heavy_check_mark:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**